README.md

```diff
@@ -1024,6 +1024,8 @@ See also [About Rust’s Machine Learning Community](https://medium.com/@autumn_
 * SQL [[sql](https://crates.io/keywords/sql)]
   * Generic
     * [launchbadge/sqlx](https://github.com/launchbadge/sqlx) - async PostgreSQL/MySQL/SQLite connection pool with strong typing support [![build badge](https://img.shields.io/github/workflow/status/launchbadge/sqlx/Rust/master?style=flat-square)](https://github.com/launchbadge/sqlx)
+  * Generic
+    * [rbatis/rbatis](https://github.com/rbatis/rbatis) - Rbatis-Flexible.Intelligent.Efficient ORM [![Build Status](https://travis-ci.org/zhuxiujia/rbatis.svg?branch=master)](https://travis-ci.org/zhuxiujia/rbatis)
```

Comment:
Can we fix the text on this a bit? e.g. removing all the "." between words? The description from the repo ("High Performance compile-time ORM") would work well as a possibility.